docs/lib/predictions/fragments/ios/escapehatch.md

```diff
@@ -1,12 +1,46 @@
-For any of the AWS services behind predictions, you can use the SDK object to get access to any methods we are not calling on your behalf by using the Escape Hatch like so:
+If your application requires calling any of the AWS Services behind the Amplify Predictions category in a manner that we aren't doing on your behalf, we provide an escape hatch so you can get a reference to that service.  For example, to get a reference to `AWSRekognition`:
 
 ```swift
-let plugin = try Amplify.Predictions.getPlugin(for: "awsPredictionsPlugin") as! AWSPredictionsPlugin
-let escapeHatch = plugin.getEscapeHatch(key: .rekognition)
-let rekognitionService = escapeHatch as! AWSRekognition
+guard let predictionsPlugin = try Amplify.Predictions.getPlugin(for: "awsPredictionsPlugin") as? AWSPredictionsPlugin else {
+    print("Unable to cast to AWSPredictionsPlugin")
+    return
+}
+
+guard let rekognitionService = predictionsPlugin.getEscapeHatch(key: .rekognition) as? AWSRekognition else {
+    print("Unable to get AWSRekognition")
+    return
+}
 
 let request = AWSRekognitionCreateCollectionRequest()
 if let request = request {
     rekognitionService.createCollection(request)
 }
```

Comment:
I think this is a useful example of why one would want to use this escape hatch! Any chance we can link to a place a developer could read up on `AWSRekognition` and friends?

Reply:
Added a section at the bottom of the page.